### PR TITLE
feat(create-vuestic): tailwind option

### LINKED ISSUE #3696: create-vuestic must check node version and require version above 16

### PR DESCRIPTION
**Pull request by m0ksem**
closes https://github.com/epicmaxco/vuestic-ui/issues/3717
closes #3696

### CODE REVIEW

**Comment by m0ksem** [comment]
> How can I test this without too much hassle?

```bash
cd packages/create-vuestic && yarn dev
```

or run build and install globally `cd packages/create-vuestic && yarn build && yarn global add ${pwd}`

**Comment by asvae** [comment]
I would at least have tailwind at the top of the list. At most - selected by default.

image

File: packages/create-vuestic/src/composables/useFiles.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+    return null
+  }
+
+  const addFile = async (path: string, content: string) => {
+    const resolvedPath = resolve(process.cwd(), projectName, path)
+
+    const dir = dirname(resolvedPath)
+
+    console.log(dir, resolvedPath)
```
Comment (asvae) [inline packages/create-vuestic/src/composables/useFiles.ts:27]:
```suggestion
```

**Comment by asvae** [comment]
Works all right. I tried to spin a project with tailwind and am seeing no issues.

Let's remove console.log and merge this.

**Review by asvae — APPROVED** [review]

**Pull request merged**